TomTomSDKExamples/Examples/UseCase/Offline/OfflineNavigationContent.swift

```diff
@@ -0,0 +1,689 @@
+//  © 2023 TomTom NV. All rights reserved.
+//
+//  This software is the proprietary copyright of TomTom NV and its subsidiaries and may be
+//  used for internal evaluation purposes or commercial use strictly subject to separate
+//  license agreement between you and TomTom NV. If you are the licensee, you are only permitted
+//  to use this software in accordance with the terms of your license agreement. If you are
+//  not the licensee, you are not authorized to use this software in any manner and should
+//  immediately return or destroy it.
+
+// System modules
+import Combine
+import CoreLocation
+import Foundation
+import SwiftUI
+
+// GO SDK modules
+import TomTomSDKCommon
+import TomTomSDKCommonUI
+// Offline
+import TomTomSDKDataManagementOffline
+import TomTomSDKDefaultTextToSpeech
+import TomTomSDKLocationProvider
+import TomTomSDKMapDisplay
+import TomTomSDKMapDisplayDataProviderOffline
+import TomTomSDKNavigation
+import TomTomSDKNavigationEngines
+import TomTomSDKNavigationUI
+import TomTomSDKRoute
+import TomTomSDKRoutePlanner
+import TomTomSDKRoutePlannerOnline
+import TomTomSDKRouteReplannerDefault
+import TomTomSDKStyleProviderOffline
+
+/// This example shows how to build a simple navigation application using the TomTom Navigation SDK for iOS.
+/// The application downloads a region, displays a map and shows the user’s location.
+/// After the user selects a destination with a long press, the app plans a route and draws it on the map.
+/// Navigation is started automatically using the route simulation.
+/// The application will display upcoming manoeuvres, remaining distance, estimated time of arrival (ETA), current speed, and speed limit information.
+struct OfflineNavigationContent: View {
+    var body: some View {
+        OfflineMainView()
+    }
+}
+
+// MARK: - OfflineNavigationController
+
+/// An observable class to handle flows between modules
+final class OfflineNavigationController: ObservableObject {
+    // MARK: Lifecycle
+
+    convenience init() {
+        let textToSpeech = SystemTextToSpeechEngine()
+        let routePlanner = TomTomSDKRoutePlannerOnline.OnlineRoutePlanner(apiKey: Keys.apiKey)
+        let routeReplanner = DefaultRouteReplanner(routePlanner: routePlanner, replanningPolicy: .findBetter)
+        let locationProvider = DefaultCLLocationProvider()
+        let simulatedLocationProvider = SimulatedLocationProvider(delay: .tt.seconds(1))
+        let navigationConfiguration = NavigationConfiguration(
+            apiKey: Keys.apiKey,
+            locationProvider: simulatedLocationProvider,
+            routeReplanner: routeReplanner,
+            betterProposalAcceptanceMode: .automatic
+        )
```

Comment:
This will not create offline navigation, but rather use the online navigation. Line 53 also creates the online route planner. 

Given that this is the OfflineNavigationController in the navigation example app, shouldn't it work on the offline map? We have a task to create factories to simplify the creation, if this can wait 2-3 weeks this should be easily addable (and hopefully not depend on feature toggles, either). Otherwise, this here should at least need a comment that it is not actually offline.

Reply:
I'd prefer to not keep this PR for so long. Let's comment or add in the README of the use case.

Reply:
It's mentioned in the README file.